packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -229,7 +226,11 @@ export const ParticipantItem = (props: ParticipantItemProps): JSX.Element => {
       data-is-focusable={true}
       data-ui-id="participant-item"
       className={mergeStyles(
-        participantItemContainerStyle({ localparticipant: me, clickable: !!menuItems }),
+        participantItemContainerStyle({
+          localparticipant: me,
+          clickable: !!menuItems,
+          showMenuButton: itemHovered || !menuHidden
```

Comment:
Why can't we hide the `menuButton` declared below instead of using class names in styles?